### PR TITLE
[SITES-381] Feature/update news articles

### PR DESCRIPTION
**Pull request by ltankey**
This lays the foundations for SITES-287. Further template changes are required to expose the summary fields to the necessary submission approval and node templates.

This PR ensures that the submission (and node creation) forms are more consistent, by exposing the necessary summary and short_summary fields through to both forms.

Short summary gets shifted into node, and summary remains in news article.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ This file is influenced by http://keepachangelog.com/.
 - Added a node type for nodes with custom templates
 - Added list of available custom template files when creating a custom template node
 - Added published_at field to all pages
-
+- All pages now have short_summary fields
+- News aritcle metadata can be edited
```
Comment (maxious) [inline CHANGELOG.md]: typo

**Comment by maxious** [comment]
LGTM

**Comment by micapam** [comment]
LGTM, taking the caveats in the description re. updating content for summary etc. (If that is proving difficult to do, happy to discuss...?)

**Comment by ltankey** [comment]
@micapam this PR now includes the ability to update the summary fields in the submission form, as well as exposing the necessary news article fields in its metadata form. I'll update the base PR message. 

Just finalising some tests.

**Comment by tobyhede** [comment]
LGTM

**Comment by ltankey** [comment]
rebased

**Pull request merged**